pages/linux/b2sum.md

```diff
@@ -0,0 +1,15 @@
+# b2sum
+
+> Calculate BLAKE2 cryptographic checksums.
+
+- Calculate the BLAKE2 checksum for a file:
+
+`b2sum {{filename1}}`
+
+- Calculate BLAKE2 checksums for multiple files:
+
+`b2sum {{filename1}} {{filename2}}`
+
+- Read a file of BLAKE2 sums and verify all files have matching checksums:
```

Comment:
We should clarify that the file contains the blake2 sum _and_ the file name as well.

Reply:
Good shout. I took this directly from the `sha1sum` page, which is apparently different to the `shasum` page - hrm.